overview.qmd

```diff
@@ -70,6 +70,12 @@ and release version of base R from the same day that the R-multiverse snapshot w
 R-multiverse is powered by [R-universe](https://r-universe.dev/), [GitHub Actions](https://github.com/features/actions),
 and the public source code repositories in the [R-multiverse GitHub organization](https://github.com/r-multiverse).
 
+## How you can help
+
+* Share R-multiverse with your colleagues.
+* [Contribute your packages](contributors.md) to R-multiverse.
+* Become a [moderator](moderators.md).
```

Comment:
```suggestion
* Volunteer to become a [moderator](moderators.md).
```

Reply:
added in 7d580a7501d7344f852ded0743ffab3e41c6f9a1